src/explore-education-statistics-frontend/src/prototypes/components/PrototypeSearchForm.tsx

```diff
@@ -1,5 +1,5 @@
 import debounce from 'lodash/debounce';
-import React, { Component, KeyboardEvent } from 'react';
+import React, {ChangeEvent, Component, KeyboardEvent} from 'react';
```

Comment:
Run Prettier before commits (should be running on precommit Git hook). Formatting problems gets picked up by the build.